.sauce/espresso.yml

```
@@ -10,23 +10,17 @@ sauce:
 defaults:
   timeout: 5m
 espresso:
-  app: ./tests/e2e/espresso/calc.apk
-  testApp: ./tests/e2e/espresso/calc-success.apk
+  app: ./tests/e2e/espresso/mda-1.0.17-20.apk
+  testApp: ./tests/e2e/espresso/mda-androidTest-1.0.17-20.apk
 suites:
   - name: "saucy barista"
-    testOptions:
-      class:
-        - com.example.android.testing.androidjunitrunnersample.CalculatorAddParameterizedTest
-        - com.example.android.testing.androidjunitrunnersample.CalculatorInstrumentationTest
+    devices:
+      - name: "Google.*"
+        platformVersion: "14"
```

Shouldn't that also be 13, same as VDC?
```suggestion
        platformVersion: "13"
```

---

Found `14` is the highest version for RDC https://saucelabs.com/products/platform-configurator

Use `14` here or use `13`?

---

I'd prefer 13, as that's the current version. [14 is beta](https://developer.android.com/about/versions/14).

---

👌